<launch>

    <arg name="multi_node" default="false"/>
    
    
    <group if="$(arg multi_node)">
        <node pkg="hgr" name="hand_detection" type="hand_detection_node.py" output="screen"/>
        <node pkg="hgr" name="hand_hand_classification" type="hand_classification_node.py" output="screen"/>
    </group>
    
    <group unless="$(arg multi_node)">
        <node pkg="hgr" name="hand_gesture_recognition" type="hand_gesture_recognition_node.py" output="screen"/>
    </group>

    
</launch>